### PR TITLE
[Params] Fixes devel portal contracts/service_plans/application controllers

### PR DESCRIPTION
**Pull request by jsmadis**
**What this PR does / why we need it**:
Fixes usage of params in the controller in the:

```
./lib/developer_portal/app/controllers/developer_portal/admin/contracts_controller.rb
./lib/developer_portal/app/controllers/developer_portal/admin/contracts/service_plans_controller.rb
./lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb
```

### CODE REVIEW

File: lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb [hunk 1]
```diff
@@ -169,14 +169,23 @@ def authorize_update_app
...
+    application_attributes.permit(*permitted_params).to_h
   end
 
   def fields_definitions
     FieldsDefinition.by_provider(site_account).by_target('Cinstance').pluck(:name)
   end
+
+  private
+
+  def local_params
```
Comment (akostadinov) [inline lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb:188]: Maybe name it `app_params` or something? Unless this `local_params` name is some convention I'm unaware of, in which case it is fine with me.

File: lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb [hunk 2]
```diff
@@ -169,14 +169,23 @@ def authorize_update_app
...
+    params.permit(%i[plan_id service_id page]).to_h
```
Comment (akostadinov) [inline lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb:189]:
```suggestion
    params.permit(%i[plan_id service_id page])
```

In file parameters seem to be used as parameters, not hash. So perhaps remove `to_h`.

**Review by akostadinov — CHANGES_REQUESTED** [review]
Looks great, just 2 nitpicks.

File: lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb [hunk 3]
```diff
@@ -115,15 +115,15 @@ def applications
   def service
     @service ||= if single_service?
                    services.first
-                 elsif (service_id = params[:service_id])
+                 elsif (service_id = local_params[:service_id])
```
[the remark this answers is not in the document]
Comment (jsmadis) [reply lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb:118]: You are right, it's not. Those PR's were made when I had insufficient knowledge in the problematic area and are made to point stuff like this. Thanks for the comment.

**Comment by jsmadis** [comment]
After further investigation, I found out that those files don't need to be changed.

**Pull request closed without merging**